### PR TITLE
lint: json.js

### PR DESCRIPTION
**Pull request by JohnMcLear**
- [x] Line 419 left to do :)

Please check I got Line 419 right :)

### CODE REVIEW

File: bin/doc/json.js [hunk 1]
```diff
@@ -246,13 +246,11 @@ const processList = (section) => {
   // depending on the section type, the list could be different things.
 
   switch (section.type) {
-    case 'ctor':
-    case 'classMethod':
```
Comment (rhansen) [inline bin/doc/json.js:250]: Why were these two lines removed? Their removal changes the behavior.

File: bin/doc/json.js [hunk 2]
```diff
@@ -486,7 +485,7 @@ const finishSection = (section, parent) => {
 // Not a general purpose deep copy.
 // But sufficient for these basic things.
 const deepCopy = (src, dest) => {
-  Object.keys(src).filter((k) => !dest.hasOwnProperty(k)).forEach((k) => {
+  Object.keys(src).filter((k) => !Object.prototype.hasOwnProperty.call(k)).forEach((k) => {
```
Comment (rhansen) [inline bin/doc/json.js:488]:
```suggestion
  Object.keys(src).filter((k) => !Object.prototype.hasOwnProperty.call(dest, k)).forEach((k) => {
```

File: bin/doc/json.js (hunk 1 above)
Comment (JohnMcLear) [reply bin/doc/json.js]: My mistake, thanks for spotting!

File: bin/doc/json.js (hunk 2 above)
Comment (JohnMcLear) [reply bin/doc/json.js]: Wow that was a big one to miss..  Can't beat 11pm sloppy work 🖌️

**Pull request merged**